src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandlerTest.java

```diff
@@ -83,4 +85,50 @@ void shouldReturnWithoutError() {
 
         assertThat(response.getErrors()).isEmpty();
     }
+
+    @Test
+    void givenSetAsideGranted_thenExcludePanelMembers() {
+        JudicialUserBase judge = new JudicialUserBase("678", "1234");
+        caseData.getPostHearing().setReviewType(PostHearingReviewType.SET_ASIDE);
+        caseData.getPostHearing().getSetAside().setAction(SetAsideActions.GRANT);
+        caseData.setHearings(List.of(Hearing.builder()
+            .value(HearingDetails.builder()
+                .panel(JudicialUserPanel.builder()
+                    .assignedTo(judge)
+                    .build())
+                .build())
+            .build()));
+
+        when(callback.getCaseDetails()).thenReturn(caseDetails);
+        when(caseDetails.getCaseData()).thenReturn(caseData);
+
+        PreSubmitCallbackResponse<SscsCaseData> response =
+                handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
+
+        assertThat(response.getErrors()).isEmpty();
+        assertThat(response.getData().getSchedulingAndListingFields().getPanelMemberExclusions().getExcludedPanelMembers().contains(new CollectionItem<>("", judge))).isTrue();
```

Comment:
Could probably chop this on to new lines and the one below.

Reply:
done